<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QWidget" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление сертификата</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Впишите данные об ученике</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Пример:</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="data_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>231</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Номер сертификата
Имя
Отчество
Фамилия
Наименование программы
Балл (хорошо, отлично и т.д)
Начало обучения
Конец обучения
Страна
Город
Площадка</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить сертификат</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_select_file">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить данные из файла</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
